contracts/Minter.vy

```diff
@@ -22,6 +23,26 @@ event Minted:
     gauge: address
     minted: uint256
 
+event UpdateMiningParameters:
+    time: uint256
+    rate: uint256
+
+# General constants
```

Comment:
can compare these constants to [ERC20CRV constants](https://github.com/curvefi/curve-dao-contracts/blob/4e428823c8ae9c0f8a669d796006fade11edb141/contracts/ERC20CRV.vy#L62)